apps/docs/pages/migrating.mdx

```diff
@@ -0,0 +1,63 @@
+# Migrating
+
+## From v1 to v2
+
+Deroll v1 is based on Cartesi Rollups v1, while deroll v2 is based on Cartesi Rollups v2.
+
+Here are the changes you should be aware of when migrating from v1 to v2:
+
+### Input metadata
+
+The input metadata received in an advance handler has changed.
+
+```diff
++chain_id: number;
++app_contract: Address;
+msg_sender: Address;
+-epoch_index: number;
+input_index: number;
+block_number: number;
+-timestamp: number;
++block_timestamp: number;
++prev_randao: Hex;
+```
+
+### Voucher value
+
+Vouchers now have a required hex-encoded bigint `value` field, which is the amount of native token to be sent during voucher execution.
```

Comment:
Maybe we could make the `value` field optional at the HTTP API level, simply to avoid these unnecessary breaking changes... I find zero a great default value, because it was exactly the value that was being passed before, and the value that will be passed in most cases, which don't deal with Ether.

Reply:
I welcome this change, but it has to be raised in the [openapi repository](https://github.com/cartesi/openapi-interfaces) and in the [implementation repository](https://github.com/cartesi/machine-emulator-tools)